bridge/src/main/java/io/stargate/bridge/service/SchemaHandler.java

```diff
@@ -154,8 +154,9 @@ private static CqlTable buildCqlTable(Table table) throws StatusException {
       }
     }
 
-    // TODO: no table options in Table?
-    // cqlTableBuilder.putOptions(...);
+    if (table.comment() != null) {
+      cqlTableBuilder.putOptions("comment", table.comment());
+    }
```

Comment:
@jeffreyscarpenter @olim7t any chance that something else was missed here? Eric now fixed the comment, but there was a `// TODO: no table options in Table?` and I am not sure if there are any other stuff that was missed..